src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/GitLabPersonalAccessTokenManagementService.java

```diff
@@ -54,8 +54,8 @@ public class GitLabPersonalAccessTokenManagementService extends VcsTokenManageme
     /**
      * The config parameter for enabling VCS access tokens.
      */
-    @Value("${artemis.version-control.version-control-access-token:#{false}}")
-    private Boolean versionControlAccessToken;
+    @Value("${artemis.version-control.version-control-access-token:#{true}}")
```

Comment:
Why to change this default value for **GitLab** here? How is that related to the localVC implementation for tokens?

Reply:
yea good point, that wasn't necessary, just edited it for testing purposes. Reverted the change